ui/src/pages/repos/[repository]/index.tsx

```diff
@@ -14,6 +14,7 @@ const RepoDetailsPage = () => {
 
   const { loading, error, data } = useQuery(GET_REPO_SYNCS, {
     variables: { id: repository },
+    pollInterval: 5000,
   })
```

Comment:
we should probably handle the `error` case here as well, yeah? It can be simple for now - maybe we could use the alert stack? https://624b441a141307004a094f09-anonmxrnpj.chromatic.com/?path=/story/utilities-alertstack--alert-stack-util